paywall/src/unlock.js/MainWindowHandler.ts

```diff
@@ -106,11 +107,10 @@ export default class MainWindowHandler {
 
   setCachedLockedState = (newState: boolean) => {
     try {
-      // this is a fast cache. The value will only be used
-      // to prevent a flash of ads on startup. If a cheeky
-      // user attempts to prevent display of ads by setting
-      // the localStorage cache, it will only work for a
-      // few milliseconds
+      // this is a fast cache. The value will only be used to prevent
+      // a flash of ads on startup. If a user attempts to prevent
+      // display of ads by setting the localStorage cache, it will
+      // only work for a few milliseconds
```

Comment:
We should extract this to a module with a reasonable name so that we can remove this comment and the use case is clear